b2share/config.py

```diff
@@ -258,7 +258,7 @@
 #: Accepted content types for Celery.
 CELERY_ACCEPT_CONTENT = ['json', 'msgpack', 'yaml']
 #: Beat schedule
-CELERYBEAT_SCHEDULE = {
+CELERY_BEAT_SCHEDULE = {
```

Comment:
You are sure about this change? It seams that the variable was [renamed in celery 4](http://docs.celeryproject.org/en/latest/userguide/configuration.html#new-lowercase-settings) but not like this, and the previous name is still supported.

Reply:
This actually seems to come from Invenio, see: 
http://pythonhosted.org/Flask-CeleryExt/#celery-v3-and-v4-configuration
https://github.com/inveniosoftware/flask-celeryext/blob/master/flask_celeryext/_mapping.py#L17
https://github.com/inveniosoftware/flask-celeryext/blob/master/flask_celeryext/app.py#L27

Reply:
Ok thanks, I missed that.